src/context/WalletProvider/MobileWallet/components/MobileTestPhrase.tsx

```diff
@@ -0,0 +1,190 @@
+      setInvalidTries([])
+      setTestCount(testCount + 1)
+    } else {
+      setInvalidTries([...invalidTries, index])
+    }
+  }
+
+  moduleLogger.info({ testState }, 'TestState')
+
+  return !testState ? null : (
+    <>
+      <ModalHeader>
+        <Text translation={'walletProvider.shapeShift.testPhrase.header'} />
+      </ModalHeader>
+      <ModalBody>
+        <RawText>
+          <Text
+            as='span'
+            color='gray.500'
+            translation={'walletProvider.shapeShift.testPhrase.body'}
+          />{' '}
+          <Tag colorScheme='green'>
+            {translate(
+              `walletProvider.shapeShift.testPhrase.${testState.targetWordIndex + 1}${ordinalSuffix(
+                testState.targetWordIndex + 1,
+              )}`,
+            )}
+            <Text as='span' ml={1} translation={'walletProvider.shapeShift.testPhrase.body2'} />
+          </Tag>{' '}
+          <Text
+            as='span'
+            color='gray.500'
+            translation={'walletProvider.shapeShift.testPhrase.body3'}
+          />
+        </RawText>
+        <Wrap mt={12} mb={6}>
+          {vault &&
+            testState &&
+            testState.randomWords.map((word: string, index: number) =>
+              revocable(
+                <Button
+                  key={index}
+                  flex='1'
+                  minW='30%'
+                  variant='ghost-filled'
+                  colorScheme={invalidTries.includes(index) ? 'gray' : 'blue'}
+                  isDisabled={invalidTries.includes(index)}
+                  onClick={() => handleClick(index)}
+                >
+                  {word}
+                </Button>,
+                revoker.addRevoker.bind(revoker),
+              ),
+            )}
+        </Wrap>
+        {/*{isLegacyWallet && (*/}
+        {/*  <Box>*/}
+        {/*    <Box position='relative' mb={8} mt={10}>*/}
+        {/*      <Divider />*/}
+        {/*      <Text*/}
+        {/*        translation={'common.or'}*/}
+        {/*        transform='translate(-50%, -50%)'*/}
+        {/*        left='50%'*/}
+        {/*        position='absolute'*/}
+        {/*        color='gray.500'*/}
+        {/*      />*/}
+        {/*    </Box>*/}
+        {/*    <Checkbox mb={4} spacing={4} onChange={onCheck} isChecked={hasAlreadySaved}>*/}
+        {/*      <Text*/}
+        {/*        fontSize='sm'*/}
+        {/*        translation={'walletProvider.shapeShift.legacy.alreadySavedConfirm'}*/}
+        {/*      />*/}
+        {/*    </Checkbox>*/}
+        {/*    <Button*/}
+        {/*      colorScheme='blue'*/}
+        {/*      width='full'*/}
+        {/*      size='md'*/}
+        {/*      isDisabled={!hasAlreadySaved}*/}
+        {/*      onClick={() => history.push('/mobile/password', { vault })}*/}
+        {/*    >*/}
+        {/*      <Text translation={'common.skip'} />*/}
+        {/*    </Button>*/}
+        {/*  </Box>*/}
+        {/*)}*/}
```

Comment:
I left this chunk here as a reference to come back to to implement importing a legacy SS wallet. I can remove it if so desired.